feature/logo

Anastasia-front opened the pull request:
add logo.svg, create Logo component and put it in the header  

cannot add hover color, tried many different options - need help

Anastasia-front committed:
add logo.svg, create Logo component and put it in the header

BogdanGlazkov commented on src/components/Logo/Logo.jsx:
```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link';
+import LogoIcon from 'public/icons/logo.svg';
+import { routes } from 'routes';
```
Відступом відокремити зовнішні імпорти від внутрішніх

Anastasia-front replied:
зробила

BogdanGlazkov commented on src/components/Logo/Logo.jsx:
```diff
@@ -0,0 +1,15 @@
+    <Link
+      href={routes.HOME}
+      aria-label="логотип"
+      className="flex items-center px-[7px] py-[6px] w-[48px] h-[48px]"
```
На десктопі розміри більші

BogdanGlazkov replied:
Додати hover, focus та перехід (напр. duration-300)

Anastasia-front replied:
зробила

BogdanGlazkov commented on src/components/Logo/Logo.jsx:
```diff
@@ -0,0 +1,15 @@
+import Link from 'next/link';
+import LogoIcon from 'public/icons/logo.svg';
```
Іконку треба або оптимізувати, щоб вона змінювала колір, прибрати fill, розміри, id, або замінити на png картинку та підміняти її при ховері

Anastasia-front replied:
зробила

Anastasia-front committed:
Merge branch 'dev' into feature/logo

Anastasia-front committed:
change logo.svg and add necessary styles to it

talk with the designer and dо as we agreed with him on the sizes - on a mobile phone and a tablet, a 36x36 logo in a 48x48 container (with paddings of 6x), on a desktop - a logo and a container of the same size 52x52

Anastasia-front committed:
delete container

Anastasia-front committed:
fix logo color

BogdanGlazkov commented on src/components/Logo/Logo.jsx:
```diff
@@ -1,15 +1,20 @@
+        aria-label="логотип"
+        className="flex items-center
+      w-[36px] h-[36px] xl:w-[52px] xl:h-[52px]
+      text-white hover:text-accent focus:text-accent duration-300"
```
Додати transition

Anastasia-front replied:
на іконках соц мереж я додавала, ти написав видалити - тут за твоєю логікою зробила вже без нього
але додала тут тепер transition
то за яким принципом його додавати?

Anastasia-front replied:
зрозуміла - Надія за це вдень написала, що transition має бути всюди де є переходи

BogdanGlazkov replied:
Так, тепер скрізь додаєм. Вимоги як макет дизайнера, постійно щось може помінятися)

Anastasia-front replied:
при фокусі виходить залипання кольору - забула за це вдень написати
так само й у іконках соціальних мереж
так має бути?

Anastasia-front committed:
add transition
